views/includes/check-your-answers/beneficial-owner-individual.html

```diff
@@ -59,7 +59,7 @@ <h3 class="govuk-heading-m">Individual beneficial owner</h3>
       actions: {
         items: [ CREATE_CHANGE_LINK(
           OE_CONFIGS.BENEFICIAL_OWNER_INDIVIDUAL_URL + "/" + boi.id + OE_CONFIGS.FIRST_NAME,
-          "Individual beneficial owner " + boi.first_name + " - first name",
+          "Individual beneficial owner " + boi.first_name + " " + boi.last_name + " - first name",
```

Comment:
It seems we have added a double quotes. Can we add unit test if that is possible

Reply:
I'll give it a try